config/example/config.txt

```diff
@@ -471,3 +471,7 @@ BYOND_ACCOUNT_AGE_THRESHOLD 7
 ## THE TRAILING SLASH ON THE END IS IMPORTANT SINCE IT JUST APPENDS THE CKEY TO THE END IN RAW
 ## Add a hash before the line below to disable the system
 CENTCOM_BAN_DB_URL https://centcom.melonmesa.com/ban/search/
+
+## This is to let the game know that the config files are loaded properly.
```

Comment:
This is the wrong way to go about this and is an unnecessary config change. Just use `if(fexists("config/config.txt"))` inside the config loader.

Reply:
I *knew* there would be an easier way to do this, thanks!

Can't put it in `load_configuration()` though since that's done before `enable_debugger()`. I want it to make a breakpoint if someone's running the game through VSCode.